config/tileset/aerial.json

```diff
@@ -428,6 +428,14 @@
       "minZoom": 13,
       "category": "Event"
     },
+    {
```

Comment:
@dwsilk  Do we also add another layer in here?

Reply:
https://github.com/linz/basemaps-config/blob/master/config/tileset/aerial.json#L896 <- this is already merged into `master`, so shouldn't be showing as a change on this PR?